### PR TITLE
Resolved issues #23 - switch to Coditsu, update all gems

### PR DESCRIPTION
**Pull request by agwozdowski**
Resolved issues #23 - switch to Coditsu with quality management, update all gems

### CODE REVIEW

**Comment by pavlo-vavruk** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,5 @@
 source 'https://rubygems.org'
 
 gem 'karafka'
-
-group :development, :test do
-  gem 'polishgeeks-dev-tools'
```
Comment (mensfeld) [inline Gemfile]: :dancer:

File: Gemfile.lock [hunk 2]
```diff
@@ -128,77 +92,41 @@ GEM
...
   ruby
 
 DEPENDENCIES
   karafka
-  polishgeeks-dev-tools
+  rspec
+  simplecov
 
 BUNDLED WITH
-   1.13.6
```
Comment (mensfeld) [inline Gemfile.lock]: removing all of this feels soooo good!

**Comment by nijikon** [comment]
Finally ;)

**Pull request merged**